js/src/components/shipping-rate-section/estimated-shipping-rates-card/rate-form-modals/validateShippingRateGroup.test.js

```diff
@@ -25,18 +25,29 @@ describe( 'validate', () => {
 			countries: [],
 		};
 
-		const errors = validate( values );
+		const errors = validateShippingRateGroup( values );
 
 		expect( errors ).toHaveProperty( 'countries' );
 	} );
 
+	it( 'has errors.rate when values.rate is empty', () => {
+		const values = {
+			...validValues,
+			rate: undefined,
+		};
```

Comment:
💅  Similar concern as https://github.com/woocommerce/google-listings-and-ads/pull/1473#discussion_r864612934

Reply:
I have made similar changes in [`1f5e14e` (#1473)](https://github.com/woocommerce/google-listings-and-ads/pull/1473/commits/1f5e14e990f5d05a915f7be0ed6678d6bd42ec90) and [`a337a1b` (#1473)](https://github.com/woocommerce/google-listings-and-ads/pull/1473/commits/a337a1b0a5fec3882833a9fc6525b47c6d27207c).